README.mediawiki

```diff
@@ -606,7 +643,8 @@ Provides linkage between OpenStackInfrastrcuture components and relevant integra
 == Changes ==
 
 ;2.2.0
-* Fix ZEN-17017: Add Port-vNIC links in UI
+* Added Cinder block storage components
```

Comment:
We should mention the following zProperties somewhere:

* zOpenStackNovaApiHosts
* zOpenStackExtraHosts
* zOpenStackCinderApiHosts